### DISCUSSION TITLE
pcp_combine to sum snow amounts

### DISCUSSION THREAD

**Discussion by craigr3** [dtcenter/METplus#2121] [Configuration] [answered]
How do I use METPLUS pcp_combine to sum up 3 hourly snow amounts to a 24 hour period?  Is there a way to change GRIB variable from APCP to SNWFL ( the grib2 code we use for snowfall).

**Comment by CPKalb** [comment, marked as answer]
Hi Craig:

[Here](https://metplus.readthedocs.io/en/latest/generated/met_tool_wrapper/PCPCombine/PCPCombine_sum.html#sphx-glr-generated-met-tool-wrapper-pcpcombine-pcpcombine-sum-py) is an example using the Pcp-Combine sum option.  If your data is observations, you would want to change everything that says "FCST" to "OBS."  To get this to work for 3 hourly snow amounts, you would to change the following:

FCST_PCP_COMBINE_INPUT_ACCUMS = 3
FCST_PCP_COMBINE_INPUT_NAMES = SNWFL

FCST_PCP_COMBINE_OUTPUT_ACCUM = 24
FCST_PCP_COMBINE_OUTPUT_NAME = whatever you want to name the output variable

You will also need to change FCST_PCP_COMBINE_INPUT_LEVELS.  However, I would need to see the output from wgrib2 to know what this variable should be set to.  Additionally, you would need to set FCST_PCP_COMBINE_INPUT_DIR to your input file directory, and FCST_PCP_COMBINE_INPUT_TEMPLATE to your input file name, with timing templates.

Christina

**Comment by craigr3** [comment]
Christina, thanks for responding so fast.  That was the info I needed.Sent from my Verizon, Samsung Galaxy smartphone
-------- Original message --------From: Christina Kalb ***@***.***> Date: 4/13/23  10:06 PM  (GMT-06:00) To: dtcenter/METplus ***@***.***> Cc: craigr3 ***@***.***>, Author ***@***.***> Subject: Re: [dtcenter/METplus] pcp_combine to sum snow amounts (Discussion #2121) 
Hi Craig:
Here is an example using the Pcp-Combine sum option.  If your data is observations, you would want to change everything that says "FCST" to "OBS."  To get this to work for 3 hourly snow amounts, you would to change the following:
FCST_PCP_COMBINE_INPUT_ACCUMS = 3
FCST_PCP_COMBINE_INPUT_NAMES = SNWFL
FCST_PCP_COMBINE_OUTPUT_ACCUM = 24
FCST_PCP_COMBINE_OUTPUT_NAME = whatever you want to name the output variable
You will also need to change FCST_PCP_COMBINE_INPUT_LEVELS.  However, I would need to see the output from wgrib/wgrib2 to know what this variable should be set to.  Additionally, you would need to set FCST_PCP_COMBINE_INPUT_DIR to your input file directory, and FCST_PCP_COMBINE_INPUT_TEMPLATE to your input file name, with timing templates.
Christina

**Reply by CPKalb** [reply]
Great.  I'm going to mark this as answered and lock it to prevent future posts.  If you have more questions, feel free to start a new discussion.

Christina
